docs/src/further_topics/ugrid/operations.rst

```diff
@@ -502,50 +518,29 @@ earlier:
         ...         start_index=face_node.start_index,
         ...     )
         ...     return mesh
-
-        >>> print(face_cube)
-        face_data / (K)                     (-- : 2; height: 3)
-            Dimension coordinates:
-                height                          -          x
-            Mesh coordinates:
-                latitude                        x          -
-                longitude                       x          -
-            Attributes:
-                Conventions                 'CF-1.7'
-
-        # Convert our mesh+data to a PolyData object.
-        # Just plotting a single height level.
-        >>> face_polydata = cube_faces_to_polydata(face_cube[:, 0])
+        ...
```

Comment:
@pp-mo You can now simplify the call to `Transform.from_unstructured` as follows:

```python
mesh = Transform.from_unstructured(
    lons.points,
    lats.points,
    connectivity=indices,
    data=cube.data,
    name=f"{cube.name()} / {cube.units}",
)
```

There is now no need to provide the `start_index` as `geovista` will detect this, and it's better to name the `indices` with the `connectivity` kwarg for clarity.